config_template.cfg

```diff
@@ -85,8 +85,6 @@ REYNOLDS_NUMBER= 6.5E6
 %
 % Reynolds length (1 m, 1 inch by default)
 REYNOLDS_LENGTH= 1.0
-
-% -------------------- INCOMPRESSIBLE FREE-STREAM DEFINITION ------------------%
 %
```

Comment:
Are we still using this?

Reply:
It has been replaced by "INCOMPRESSIBLE FLOW CONDITION DEFINITION" instead, since we are more typically dealing with internal flows. The initialization no longer uses "FREESTREAM" values, rather we simply have *_INIT values.